vendors/watteco/drivers/hygrotemp'o/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "hygrotempo_v4",
+  "version": "1.0.1",
```

Is not it the first driver for watteco hygrotempo? the version should be 1.0.0 I think.
```suggestion
  "version": "1.0.0",
```